files/fr/learn/tools_and_testing/client-side_javascript_frameworks/vue_getting_started/index.md

```diff
@@ -318,3 +318,12 @@ In the next article we'll build our first custom component, and look at some imp
   - [Working with Svelte stores](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_stores)
   - [TypeScript support in Svelte](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_TypeScript)
   - [Deployment and next steps](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_deployment_next)
+
+- Angular
+
+  - [Getting started with Angular](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Angular_getting_started)
+  - [Beginning our Angular todo list app](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Angular_todo_list_beginning)
+  - [Styling our Angular app](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Angular_styling)
+  - [Creating an item component](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Angular_item_component)
+  - [Filtering our to-do items](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Angular_filtering)
+  - [Building Angular applications and further resources](/fr/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Angular_building)
```

Comment:
```suggestion

```
This doesn't seem related (if needed I can help translate the text in French)

Reply:
No not related, in this file there was just a lot of the issues in the untranslated parts, so the easiest wasy was to just sync in the current en-US verion and change back the links to `/fr/`. I'll take the suggestion to drop this chunk though